cpp/test/test_arrow_chunk_reader.cc

```diff
@@ -34,8 +34,7 @@ namespace graphar {
 
 TEST_CASE_METHOD(GlobalFixture, "ArrowChunkReader") {
   // read file and construct graph info
-  std::string path =
-      test_data_dir + "/ldbc_sample/parquet/ldbc_sample.graph.yml";
+  std::string path = test_data_dir + "/ldbc_sample/LdbcSample.graph.yml";
```

Comment:
You can add a test case to use the json test data, it's better not modify the current test case. like:

TEST_CASE_METHOD(GlobalFixture, "TestJson") {

}

Reply:
done